app/component/CarLeg.js

```diff
@@ -42,10 +54,13 @@ function CarLeg(props, context) {
         </div>
         <div className="itinerary-leg-action">
           <FormattedMessage
-            id="car-distance-duration"
+            id={CarLeg.getTranslationKey(props.leg.mode)}
             values={{ distance, duration }}
             defaultMessage="Drive {distance} ({duration})}"
           />
+          {CarLeg.createBookButton(props.leg)}
+          <Icon img={carpoolAgencyIcon[0]} className="carpool-agency-logo" />
+          <Icon img={carpoolAgencyIcon[1]} className="carpool-agency-logo" />
```

Comment:
Mifaz has only one icon. We should not generate an empty img tag here in this case.